praw/models/reddit/subreddit.py

```diff
@@ -2982,6 +3013,15 @@ def __init__(self, subreddit):
 
         """
         self.subreddit = subreddit
+        super().__init__(subreddit._reddit, None)
+        self.subreddit_display_name = self.subreddit.display_name
```

Comment:
Is this an actual attribute returned by Reddit? If not, please prefix with `_`.

Reply:
I'll remove it altogether.